pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -48,19 +50,23 @@ import (
 )
 
 const (
-	DataMountPath             = "/opt/infinispan/server/data"
-	DataMountVolume           = "data-volume"
-	CustomLibrariesMountPath  = "/opt/infinispan/server/lib/custom-libraries"
-	CustomLibrariesVolumeName = "custom-libraries"
-	EncryptMountPath          = "/etc/encrypt"
-	ConfigVolumeName          = "config-volume"
-	EncryptVolumeName         = "encrypt-volume"
-	IdentitiesVolumeName      = "identities-volume"
-	AdminIdentitiesVolumeName = "admin-identities-volume"
+	DataMountPath                    = "/opt/infinispan/server/data"
+	DataMountVolume                  = "data-volume"
+	CustomLibrariesMountPath         = "/opt/infinispan/server/lib/custom-libraries"
+	CustomLibrariesVolumeName        = "custom-libraries"
+	EncryptMountPath                 = "/etc/encrypt"
+	ConfigVolumeName                 = "config-volume"
+	EncryptVolumeName                = "encrypt-volume"
+	IdentitiesVolumeName             = "identities-volume"
+	AdminIdentitiesVolumeName        = "admin-identities-volume"
+	EventReasonPrelimChecksFailed    = eventlog.ReasonType("PrelimChecksFailed")
```

Comment:
probably following syntax a bit clear
`EventReasonPrelimChecksFailed eventlog.ReasonType = "PrelimChecksFailed"`

Reply:
I'll remove the `ReasonType` for now